docs/reference/remap/functions/add_datadog_tags.cue

```diff
@@ -0,0 +1,45 @@
+package metadata
+
+remap: functions: add_datadog_tags: {
+	category: "String"
+	description: #"""
+		Add a set of tags to an existing tags list following the Datadog tag format. Datadog logs tags should be kept in the `ddtags` fields to be properly accounted for if the
+		event is ultimately sent to Datadog using the `datadog_logs` sink. Duplicated tags are removed.
```

Comment:
```suggestion
		event is ultimately sent to Datadog using the `datadog_logs` sink. Duplicated tags are ignored.
```

Maybe? Removed might be interpreted as "if there are any duplicated tags they are all removed" rather than "only the single tag is preserved" 🤷